react-carousel-component/my-app/src/Carousel.js

```diff
@@ -0,0 +1,57 @@
+import { FaChevronLeft, FaChevronRight, FaRegCircle, FaCircle } from 'react-icons/fa';
+import { useEffect, useState } from 'react';
+import './Carousel.css';
+
+/**
+ * Carousel cycles through images automatically and allows users to browse manually.
+ * @param {images} images represents an array of objects. Each object will contain an image URL and alt text.
+ * @returns
```

Comment:
I usually leave off `@returns` on React Components because it's obvious that they return a JSX Element.